client/my-sites/site-settings/manage-connection/ownership-information.jsx

```diff
@@ -0,0 +1,60 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import HappychatButton from 'components/happychat/button';
+
+const OwnershipInformation = ( { translate } ) => (
+	<FormFieldset className="manage-connection__formfieldset has-divider is-top-only">
+		<div className="manage-connection__ownership-info">
+			<Gridicon
+				icon="info-outline"
+				size={ 24 }
+				className="manage-connection__ownership-info-icon"
+			/>
+
+			<div className="manage-connection__ownership-info-text">
+				<FormSettingExplanation>
+					{ translate(
+						'{{strong}}Site owners{{/strong}} are users who have connected Jetpack to WordPress.com.',
+						{
+							components: {
+								strong: <strong />,
+							},
+						}
+					) }
+					<br />
+					{ translate(
+						'{{strong}}Plan purchasers{{/strong}} are users who purchased a paid plan for the site.',
+						{
+							components: {
+								strong: <strong />,
+							},
+						}
+					) }
+				</FormSettingExplanation>
+				<FormSettingExplanation>
+					{ translate(
+						'Usually these are the same person, but sometimes they can differ. E.g., developers may be ' +
+							'a Site owner, because they set up the website and connected Jetpack to WordPress.com and ' +
+							'their clients may be Plan purchasers who use their billing information to purchase the plan ' +
+							'for the site.'
+					) }
+					<HappychatButton>{ translate( 'Need help? Chat with us' ) }</HappychatButton>
```

Comment:
This likely needs a line break and or whitespace. I've shared a capture.

Reply:
👍 I've addressed that in fa08543fc3b0fc8aa7db18061f2aa3cca47f80cf

See the updated preview:

![](https://cldup.com/fqNIidYtZI.png)